### PR TITLE
Fix try viam link

### PR DESCRIPTION
**Pull request by mestcihazal**
Fix try viam link

### CODE REVIEW

File: docs/_index.md [hunk 1]
```diff
@@ -22,48 +22,48 @@ outputs:
...
 <div class="col hover-card link-with-icon">
-  <a href="use-cases/configure/" class="noanchor"><div>
+  <a href="/get-started/try-viam/" class="noanchor"><div>
     <picture><img src="icons/rover.svg" alt="Rover" loading="lazy"></picture>
     <p>Try Viam</p></div>
   </a>
 </div>
 <div class="col hover-card link-with-icon">
-  <a href="use-cases/configure/" class="noanchor"><div>
+  <a href="/use-cases/configure/" class="noanchor"><div>
```
Comment (npentrel) [inline docs/_index.md:31]: why did you add all the leading slashes? They do mean things on staging break unfortunately, which is why we don't use them in html

File: docs/_index.md [hunk 2]
```diff
@@ -22,48 +22,48 @@ outputs:
 <div class="cards max-page use-cases">
   <div class="row">
 <div class="col hover-card link-with-icon">
-  <a href="use-cases/configure/" class="noanchor"><div>
+  <a href="/get-started/try-viam/" class="noanchor"><div>
```
Comment (npentrel) [inline docs/_index.md:25]:
```suggestion
  <a href="get-started/try-viam/" class="noanchor"><div>
```

File: docs/_index.md (hunk 1 above)
Comment (mestcihazal) [reply docs/_index.md:31]: oh ok, didn't know that

File: docs/_index.md [hunk 3]
```diff
@@ -22,48 +22,48 @@ outputs:
...
 <div class="col hover-card link-with-icon">
-  <a href="use-cases/configure/" class="noanchor"><div>
+  <a href="/use-cases/configure/" class="noanchor"><div>
     <picture><img src="icons/build.svg" alt="Build" loading="lazy"></picture>
     <p>Build simple smart machines</p></div>
       </a>
 </div>
 <div class="col hover-card link-with-icon">
-  <a href="use-cases/image-data/" class="noanchor"><div>
+  <a href="/use-cases/image-data/" class="noanchor"><div>
```
Comment (mestcihazal) [inline docs/_index.md:37]:
```suggestion
  <a href="use-cases/image-data/" class="noanchor"><div>
```

File: docs/_index.md [hunk 4]
```diff
@@ -22,48 +22,48 @@ outputs:
...
     <picture><img src="services/icons/data-capture.svg" alt="Data Management" loading="lazy"></picture>
     <p>Capture and sync image data</p></div>
       </a>
 </div>
 <div class="col hover-card link-with-icon">
-  <a href="use-cases/sensor-data/" class="noanchor"><div>
+  <a href="/use-cases/sensor-data/" class="noanchor"><div>
```
Comment (mestcihazal) [inline docs/_index.md:43]:
```suggestion
  <a href="use-cases/sensor-data/" class="noanchor"><div>
```

File: docs/_index.md [hunk 5]
```diff
@@ -22,48 +22,48 @@ outputs:
...
     <picture><img src="icons/data-analytics.svg" alt="Data Analytics" loading="lazy"></picture>
     <p>Capture and query sensor data</p></div>
       </a>
 </div>
 <div class="col hover-card link-with-icon">
-  <a href="use-cases/deploy-code/" class="noanchor"><div>
+  <a href="/use-cases/deploy-code/" class="noanchor"><div>
```
Comment (mestcihazal) [inline docs/_index.md:49]:
```suggestion
  <a href="use-cases/deploy-code/" class="noanchor"><div>
```

File: docs/_index.md [hunk 6]
```diff
@@ -22,48 +22,48 @@ outputs:
...
     <picture><img src="icons/code-deployment.svg" alt="Code deployment" loading="lazy"></picture>
     <p>Manage and deploy code versions</p></div>
       </a>
 </div>
 <div class="col hover-card link-with-icon">
-  <a href="use-cases/deploy-ml/" class="noanchor"><div>
+  <a href="/use-cases/deploy-ml/" class="noanchor"><div>
```
Comment (mestcihazal) [inline docs/_index.md:55]:
```suggestion
  <a href="use-cases/deploy-ml/" class="noanchor"><div>
```

File: docs/_index.md [hunk 7]
```diff
@@ -22,48 +22,48 @@ outputs:
...
     <picture><img src="icons/ml.svg" alt="Machine Learning" loading="lazy"></picture>
     <p>Train and deploy classification models</p></div>
       </a>
 </div>
 <div class="col hover-card link-with-icon">
-  <a href="use-cases/navigate/" class="noanchor"><div>
+  <a href="/use-cases/navigate/" class="noanchor"><div>
```
Comment (mestcihazal) [inline docs/_index.md:61]:
```suggestion
  <a href="use-cases/navigate/" class="noanchor"><div>
```

File: docs/_index.md [hunk 8]
```diff
@@ -22,48 +22,48 @@ outputs:
...
     <picture><img src="icons/mobility.svg" alt="Mobility" loading="lazy"></picture>
     <p>Teleoperate and navigate with rovers</p></div></a>
   </div>
   <div class="col hover-card link-with-icon">
-    <a href="use-cases/manage-fleet/" class="noanchor"><div>
+    <a href="/use-cases/manage-fleet/" class="noanchor"><div>
```
Comment (mestcihazal) [inline docs/_index.md:66]:
```suggestion
    <a href="use-cases/manage-fleet/" class="noanchor"><div>
```

File: docs/_index.md [hunk 9]
```diff
@@ -86,26 +86,26 @@ outputs:
 <div class="cards max-page use-cases">
   <div class="row">
 <div class="col hover-card link-with-icon">
-  <a href="get-started/viam/" class="noanchor"><div>
+  <a href="/get-started/viam/" class="noanchor"><div>
```
Comment (mestcihazal) [inline docs/_index.md:89]:
```suggestion
  <a href="get-started/viam/" class="noanchor"><div>
```

File: docs/_index.md [hunk 10]
```diff
@@ -86,26 +86,26 @@ outputs:
...
     <picture><img src="icons/viam.svg" alt="Viam" loading="lazy"></picture>
     <p>Viam in 3 minutes</p>
   </div>
       </a>
 </div>
 <div class="col hover-card link-with-icon">
-  <a href="platform/" class="noanchor"><div>
+  <a href="/platform/" class="noanchor"><div>
```
Comment (mestcihazal) [inline docs/_index.md:96]:
```suggestion
  <a href="platform/" class="noanchor"><div>
```

File: docs/_index.md [hunk 11]
```diff
@@ -86,26 +86,26 @@ outputs:
...
     <picture><img src="icons/core.svg" alt="core" loading="lazy"></picture>
     <p>Viam Platform</p></div>
       </a>
 </div>
 <div class="col hover-card link-with-icon">
-  <a href="sdks/" class="noanchor"><div>
+  <a href="/sdks/" class="noanchor"><div>
```
Comment (mestcihazal) [inline docs/_index.md:102]:
```suggestion
  <a href="sdks/" class="noanchor"><div>
```

File: docs/_index.md [hunk 12]
```diff
@@ -86,26 +86,26 @@ outputs:
...
     <picture><img src="icons/documentation.svg" alt="Documentation" loading="lazy"></picture>
     <p>SDKs</p></div>
       </a>
 </div>
 <div class="col hover-card link-with-icon">
-  <a href="tutorials/" class="noanchor"><div>
+  <a href="/tutorials/" class="noanchor"><div>
```
Comment (mestcihazal) [inline docs/_index.md:108]:
```suggestion
  <a href="tutorials/" class="noanchor"><div>
```

File: docs/_index.md (hunk 1 above)
Comment (mestcihazal) [reply docs/_index.md:31]: fixed!

File: docs/_index.md [hunk 13]
```diff
@@ -22,13 +22,13 @@ outputs:
...
 <div class="col hover-card link-with-icon">
-  <a href="use-cases/configure/" class="noanchor"><div>
+  <a href="get-started/try-viam/" class="noanchor"><div>
     <picture><img src="icons/rover.svg" alt="Rover" loading="lazy"></picture>
     <p>Try Viam</p></div>
   </a>
 </div>
 <div class="col hover-card link-with-icon">
-  <a href="use-cases/configure/" class="noanchor"><div>
+  <a href="/use-cases/configure/" class="noanchor"><div>
```
Comment (mestcihazal) [inline docs/_index.md:31]:
```suggestion
  <a href="use-cases/configure/" class="noanchor"><div>
```

**Review by npentrel — APPROVED** [review]

**Comment by npentrel** [comment]
Thank you! Ready to merge!
